yhondanilo/chess.rb

```diff
@@ -0,0 +1,46 @@
+class Chess
+  attr_accessor :fila, :columna
+
+  def initialize(fila, columna)
+    @fila = fila
+    @columna = columna
+    intercalar
+  end
+
+  def intercalar
+    @board=Array.new(@fila){ Array.new(@columna) }
+    @board[0][0] = 1
+    for f in 0..self.fila-1
+      for c in 0..self.columna-1
+        if @board[f][c-1]==0
+          @board[f][c] =  1
+        elsif @board[f][c-1]==1
```

Comment:
Surrounding space missing for operator '=='.<br>Surrounding space missing for operator '-'.